examples/buildpacks/dual-storage-backend.md

```diff
@@ -0,0 +1,138 @@
+# Dual storage backend setup
+
+## Use cases
+
+### Migration
+
+An organization might already have a provenance storage set up for a provider,
+e.g. oci, and they want to move to document storage docdb. To ease the
+transition, they might want to output to both simultaneously during that
+transition/migration period.
+
+### Duplication due to requirements, regulated or otherwise
+
+An organization's security policy or regulatory requirements may dictate the
+data needs to be stored within multiple systems be it for redundancy or
+different methods of analysis/usage:
+
+* e.g. docdb for querying, oci for keeping the provenance next to signatures,
+  or gcs for longer term archive storage
+* if there is an issue with one of the storage methods either due to chains or
+  an outage with that particular method of storage
+
+## Setup
+
+Assuming you don't have a Kubernetes cluster available, start by spinning one
+up. We provide a simple command to provision minikube. Run the following command
+from the root of the SSF project:
+
+```bash
+make setup-minikube
+```
+
+Setup tekton and chains:
+
+> **REMARK: this tutorial requires chains 0.7+. Since it has not yet been
+released at the time of writting, it will be necessary to build the controller
+from source instead of using the task from the Makefile:**
+>
+> ```bash
+>  # Install Tekton only.
+>  ./platform/10-tekton-setup.sh
+>
+> # Go to the chains repository.
+> export KO_DOCKER_REPO=<YOUR_DOCKER_ID>
```

Comment:
Should there be more detail that this is your Docker Hub ID. Might be confusing otherwise.

Reply:
Good idea 👍 Addressed ✅